src/App.js

```diff
@@ -1,23 +1,73 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import contactsData from "./contacts.json";
+import { useState } from "react";
 
 function App() {
+  const [contacts, setContacts] = useState(contactsData.slice(0, 5)); 
+
+  const addRandomContact = () => {
```

Comment:
For that one maybe you could limit if you don't have anymore contacts to add, because otherwise this breaks your application